src/psyclone/tests/line_length_test.py

```diff
@@ -384,6 +384,20 @@ def test_long_lines_allocate(fortran_reader, fortran_writer):
     assert correct in out
 
 
+def test_long_lines_indentation():
+    '''Test that if we have too much initial indentation that we still
+    can output a result.'''
+    long_string = '''                                                     \
```

Comment:
Please could you extend this test to add a case where, even after `ltrim`, the line needs to be broken.

Reply:
Added